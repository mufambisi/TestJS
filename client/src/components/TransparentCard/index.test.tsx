import React from "react";
import { render } from "@origin-digital/ods-testing-library";
import { TransparentCard } from "./index";

const renderTransparentCard = () =>
  render(<TransparentCard>Hello world</TransparentCard>);

describe("<TransparentCard />", () => {
  it("should render without error", () => {
    const { container } = renderTransparentCard();

    expect(container).toMatchSnapshot();
  });
});
