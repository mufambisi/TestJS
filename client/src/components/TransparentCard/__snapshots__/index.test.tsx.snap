// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TransparentCard /> should render without error 1`] = `
.c0 {
  margin: 0;
  padding: 0;
  border: 0;
  box-sizing: border-box;
  font-size: 100%;
  font: inherit;
  vertical-align: baseline;
  background-color: transparent;
  padding-top: 8px;
  padding-bottom: 8px;
  width: 100%;
}

@media screen and (min-width:576px) {
  .c0 {
    padding-top: 8px;
  }
}

@media screen and (min-width:768px) {
  .c0 {
    padding-top: 8px;
  }
}

@media screen and (min-width:992px) {
  .c0 {
    padding-top: 0;
  }
}

@media screen and (min-width:1200px) {
  .c0 {
    padding-top: 0;
  }
}

@media screen and (min-width:576px) {
  .c0 {
    padding-bottom: 8px;
  }
}

@media screen and (min-width:768px) {
  .c0 {
    padding-bottom: 8px;
  }
}

@media screen and (min-width:992px) {
  .c0 {
    padding-bottom: 0;
  }
}

@media screen and (min-width:1200px) {
  .c0 {
    padding-bottom: 0;
  }
}

<div>
  <div
    class="c0"
    width="full"
  >
    Hello world
  </div>
</div>
`;
